projects/m4/012-repeated-words/js/index.js

```diff
@@ -0,0 +1,48 @@
+//012
+
+//packages
+const fs = require('fs').promises;
+
+const file = 'input.txt'
+
+//function to read the content of the file
+const readFile = file => {
+    return data = fs.readFile(file, {encoding:'utf-8'})
+    .then((data) => {
+        rowData = data.split('\n')
+        return  console.log(checkRepeatedWords(rowData))  })
```

Comment:
No Best Practice.
Return rowData.

Reply:
what is not a best practice? to call a function in the return?